barretenberg/cpp/src/barretenberg/honk/composer/ultra_transcript.test.cpp

```diff
@@ -220,38 +220,3 @@ TEST_F(UltraTranscriptTests, StructureTest)
     prover.transcript.deserialize_full_transcript();
     EXPECT_EQ(static_cast<Flavor::Commitment>(prover.transcript.sorted_accum_comm), one_group_val * rand_val);
 }
-
-TEST_F(UltraTranscriptTests, FoldingManifestTest)
```

Comment:
Until we handle alpha properly, this test doesn't really make sense so I decided to remove it.